platform-dom/src/main/java/com/softicar/platform/dom/elements/DomTextArea.java

```diff
@@ -102,15 +47,10 @@ public DomTextArea setColCount(int col) {
 		return this;
 	}
 
-	public DomTextArea setReadonly() {
-
-		setAttribute("readonly", "readonly");
-		return this;
-	}
-
 	/**
-	 * Enables/Disables wrapping behavior of text area fields. WARNING: This is
-	 * a non-standard attribute but it works in most browsers.
+	 * Toggles the wrapping behavior of this {@link DomTextArea}.
+	 * <p>
+	 * WARNING: This is a non-standard attribute but it works in most browsers.
```

Comment:
Uhm... no. It _is_ a standard attribute - we just use non-standard values, for whatever reason.
cf.:
- https://www.w3schools.com/tags/att_textarea_wrap.asp
- https://caniuse.com/mdn-html_elements_textarea_wrap

We should probably change this to either `hard` or `soft` instead of `on`, and to unset the attribute instead of setting it to `off`.

Reply:
I don't know. Then the boolean flag does not make sense. Since this method was not use here an in EAS, I removed it.